pages/post/[post].tsx

```diff
@@ -0,0 +1,149 @@
+import { fetchEntries } from "../../utils/contentfulPosts";
+import { Asset } from "contentful";
+
+interface IContent {
+    title: string
+    category: string
+    publishedDate: string
+    featuredImage: Asset
+    imageAlt: string
+    imageUrl: string
+    slug: string
+    author: string
+    body: any
+}
+
+interface HomeProps {
+    content: IContent
+}
+
+interface IBody {
+    data: {}
+    marks: {}
+    nodeType: string
+    value: string
+}
+
+export default function Home({ content }: HomeProps){
+
+    console.log('content', content.body.content[0].content[0]);
+
+    const { title, category, publishedDate, author } = content;
+    const imageAlt = content.featuredImage.fields.title;
+    const imageUrl = content.featuredImage.fields.file.url;
```

Comment:
If you reuse the `Post` interface, these lines would not be needed since you will receive `imageAlt` and `imageUrl` as props.